ecp5/bitstream.cc

```diff
@@ -915,6 +915,24 @@ void write_bitstream(Context *ctx, std::string base_config_file, std::string tex
                 static bool drive_3v3_warning_done = false;
                 if (iotype == "LVCMOS33") {
                     cc.tiles[pio_tile].add_enum(pio + ".DRIVE", str_or_default(ci->attrs, ctx->id("DRIVE"), "8"));
+                } else if (iotype == "LVCMOS33D") {
+                    if (bel.location.y == 0) {
+                        // Pseudo differential top IO
+                        NPNR_ASSERT(dir == "OUTPUT");
+                        NPNR_ASSERT(pio == "PIOA");
+                        std::string cpio_tile = get_comp_pio_tile(ctx, bel);
+                        cc.tiles[cpio_tile].add_enum("PIOB.DRIVE", str_or_default(ci->attrs, ctx->id("DRIVE"), "12"));
```

Comment:
You need to set DRIVE here in both the PIOA and PIOB tiles

Reply:
Doh, of course. Thanks